components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -131,30 +135,56 @@ export class Workspaces extends React.Component<WorkspacesProps, WorkspacesState
                         <div className="px-6 py-3 flex justify-between space-x-2 text-sm text-gray-400 border-t border-gray-200 h-96">
                             <div className="flex flex-col items-center w-96 m-auto">
                                 <h3 className="text-center pb-3">No Active Workspaces</h3>
-                                <div className="text-center pb-6 text-gray-500">Prefix a git repository URL with gitpod.io/# or open a workspace template. <a className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600" href="https://www.gitpod.io/docs/getting-started/">Learn how to get started</a></div>
-                                <button onClick={toggleTemplateModal} className="font-medium">Select Template</button>
+                                <div className="text-center pb-6 text-gray-500">Prefix any git repository URL with gitpod.io/# or start a workspace for a recently used project. <a className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600" href="https://www.gitpod.io/docs/getting-started/">Learn how to get started</a></div>
+                                <button onClick={toggleStartWSModal} className="font-medium">Start Workspace</button>
```

Comment:
**suggestion**: Using a similar voice and tone as the rest of the product sounds good. WDYT?

```suggestion
                                <button onClick={toggleStartWSModal} className="font-medium">New Workspace</button>
```